README.md

```diff
@@ -236,7 +236,7 @@ Production capacities are centralized in the [zones.json](https://github.com/tmr
 - Denmark (DK1 and DK2): [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
 - Denmark (Bornholm)
   - Wind: [stateofgreen.com](https://stateofgreen.com/en/profiles/regional-municipality-of-bornholm/solutions/kalby-wind-turbines)
-- Dominican Republic: [Climatescope](http://global-climatescope.org/en/country/dominican-republic/#/details)
```

Comment:
Can we keep this reference in place? You can add the new one below with a tagline, just because this one will be updated whereas the dropbox link is static, and has more potential to be invalid in the future

Reply:
@nessie2013 The following URL can be used as a permanent reference for the Dropbox file link your previously provided: https://archive.org/details/oc-2020-000190-gg-sie-informe-anual-2019-v-1